Fix misc. source comment typos

luzpaz opened the pull request:
Found using `codespell -q 3 -S ./thirdparty,*.po -L ang,ba,cas,dof,doubleclick,fave,hist,leapyear,lod,nd,numer,ois,paket,seeked,sinc,switchs,te,uint -D ~/Projects/codespell/codespell_lib/data/dictionary.txt `

clayjohn commented on modules/gdscript/gdscript.cpp:
```diff
@@ -99,7 +99,7 @@ GDScriptInstance *GDScript::_create_instance(const Variant **p_args, int p_argco
 	instance->owner->set_script_instance(instance);
 
-	/* STEP 2, INITIALIZE AND CONSRTUCT */
+	/* STEP 2, INITIALIZE AND CONSTRUCTT */
```
Still wrong here

luzpaz replied:
8b30bb9

clayjohn commented on modules/gdscript/gdscript_compiler.cpp:
```diff
@@ -2064,7 +2064,7 @@ Error GDScriptCompiler::_parse_class_blocks(GDScript *p_script, const GDScriptPa
 					instance->owner->set_script_instance(instance);
 
-					/* STEP 2, INITIALIZE AND CONSRTUCT */
+					/* STEP 2, INITIALIZE AND CONSTRUCTT */
```
Same as above

luzpaz replied:
8b30bb9

akien-mga commented:
Looks good, but commits should be squashed into one. See [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase).

luzpaz committed:
Fix misc. source comment typos

Found using `codespell -q 3 -S ./thirdparty,*.po -L ang,ba,cas,dof,doubleclick,fave,hist,leapyear,lod,nd,numer,ois,paket,seeked,sinc,switchs,te,uint -D ~/Projects/codespell/codespell_lib/data/dictionary.txt `

luzpaz commented:
Squashed.

akien-mga commented:
Thanks!
